lib/decidim/internal_evaluation/proposals_component.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Works when defined in decidim/decidim-proposals/lib/decidim/proposals/component.rb
+
+Decidim.register_component(:proposals) do |component|
+  component.exports :internal_evaluations do |exports|
+    exports.collection do |component_instance, user|
+      Decidim::InternalEvaluation::Export.internal_evaluations_for_resource(
+        Decidim::Proposals::Proposal, component_instance, user
+      )
+    end
+
+    exports.include_in_open_data = true
+
+    exports.serializer Decidim::InternalEvaluation::InternalEvaluationSerializer
+  end
+end
```

Comment:
@entantoencuanto this is not working, do you know how to extend a component from a module so I can define a new export?

Reply:
Maybe we should do something like decidim_awesome module does from the engine to open the proposals manifest and change the it 
https://github.com/decidim-ice/decidim-module-decidim_awesome/blob/058af7db47737e3ca108ac8e08efd5ec55d67a44/lib/decidim/decidim_awesome/engine.rb#L104-L122